src/workloads/execution/SlidingWindows.yml

```diff
@@ -10,11 +10,11 @@ Actors:
     Database: &db test
     Threads: 1
     CollectionCount: 1
-    DocumentCount: 100000
-    BatchSize: 10000
+    DocumentCount: 10000
```

Comment:
Can you drop a comment here that this is chosen specifically to avoid spilling to disk?